src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -504,13 +504,7 @@ public void setPluginManager(PluginManager pluginManager) {
      * that we need for testing.
      */
     protected ServletContext createWebServer() throws Exception {
-        server = new Server(new ThreadPoolImpl(new ThreadPoolExecutor(10, 10, 10L, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(),new ThreadFactory() {
-            public Thread newThread(Runnable r) {
-                Thread t = new Thread(r);
-                t.setName("Jetty Thread Pool");
```

Comment:
we can not apply naming with the [`QueuedThreadPool`](https://github.com/eclipse/jetty.project/blob/fedc7c65997d433bbdfc26fb3d861f8488f9c804/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java#L129) without also specifiying the number of threads etc.